Index: tmate-1.8.10/Makefile.am
===================================================================
--- tmate-1.8.10.orig/Makefile.am
+++ tmate-1.8.10/Makefile.am
@@ -22,7 +22,6 @@ CFLAGS += -D_GNU_SOURCE
 endif
 
 CFLAGS += -Wno-unused-parameter -Wno-unused-variable
-CFLAGS += -Ilibssh/include/ -Imsgpack/src
 CFLAGS += -rdynamic # for stack traces
 
 # Set flags for gcc. gcc4 whines abouts silly stuff so it needs slightly
@@ -242,15 +241,4 @@ nodist_tmate_SOURCES += compat/b64_ntop.
 endif
 
 tmate_LDADD = \
-	libssh/build/src/libssh.a \
-	msgpack/src/.libs/libmsgpackc.a
-
-*.c: $(tmate_LDADD)
-
-libssh/build/src/libssh.a:
-	cd libssh/build && cmake .. -DWITH_SFTP=OFF -DWITH_SERVER=OFF -DWITH_PCAP=OFF -DWITH_STATIC_LIB=ON -DWITH_GSSAPI=OFF
-	+make -C libssh/build ssh_static
-
-msgpack/src/.libs/libmsgpackc.a:
-	cd msgpack && ./bootstrap && ./configure
-	+make -C msgpack/src libmsgpackc.la
+	-lssh -lmsgpackc
